src/libprojectM/PresetFactoryManager.cpp

```diff
@@ -45,13 +44,11 @@ void PresetFactoryManager::initialize(int gx, int gy) {
 	}
 	  
 	PresetFactory * factory;
-	
-	#ifndef DISABLE_MILKDROP_PRESETS
```

Comment:
why was this even here lol

Reply:
Definitely a good question. Define this macro and projectM is simply useless...

Probably someone once had the idea to make the library more modular, having multiple types of presets which could be enabled or disabled independently.